wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,37 @@ permalink: "/wiki/Running-a-Server"
 </details>
 
 
-
 ## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use Servers listed by the built in Directories, at least at first while getting your sound set up with the Jamulus Client. You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.  
-
-Note that you can have a "private" session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
-
-If you decide you cannot use any of the Servers listed by the built in Directories, you may be able to use a Server (either Registered in a Custom Directory, or Unregistered - see [Server Types](#server-types)) hosted by a third party. Doing so will save you the trouble of setting one up yourself.
+**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a server or choose a third party hosting service to set up a Jamulus Server for you. If you just want an undisturbed session, use the [soloing technique described on the Tips and Tricks page](tips-tricks-more#have-a-undisturbed-session-on-any-server).
```

Comment:
> or choose a third party hosting service

"or you can choose a third party hosting service".

Reply:
> If you just want an undisturbed session, ...

I'm not really comfortable having this here, still.  It still feels too much like sanctioning what I see as bad behaviour.